googleapiclient/model.py

```diff
@@ -258,10 +258,8 @@ def serialize(self, body_value):
     return json.dumps(body_value)
 
   def deserialize(self, content):
-    try:
-        content = content.decode('utf-8')
-    except AttributeError:
-        pass
+    if isinstance(content, bytes):
+      content = content.decode('utf-8')
```

Comment:
docstring in `BaseModel.deserialize()` says type of content is string.
I think this project (or google) should decide what 'string' means.

My proposal is:

* string -- basestring in PY2, str in PY3
* bytes -- str(=bytes) in PY2, bytes in PY3
* unicode -- unicode in PY2, str in PY3